benchmarks/cugraph-dgl/pytest-based/bench_cugraph_dgl_uniform_neighbor_sample.py

```diff
@@ -0,0 +1,244 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from dask_cuda import LocalCUDACluster
+from dask.distributed import Client
+import dask_cudf
+
+
+import time
+import os
+import pytest
+import numpy as np
+import cupy as cp
+# Facing issues with rapids-pytest-benchmark plugin
+# pytest-benchmark.
+import pytest_benchmark
+benchmark = pytest_benchmark.plugin.benchmark
```

Comment:
This shouldn't be necessary. `pytest` will always make the `benchmark` fixture available if the plugin is installed.

Reply:
Resolved by https://github.com/rapidsai/cugraph/pull/3092/commits/459a190d25966daa9239b9c6a4eb510ceec34aa5